integration_tests/integration/approveLicence.cy.ts

```diff
@@ -22,7 +22,7 @@ context('Approve a licence', () => {
     })
 
     cy.task('stubGetCompletedLicence', { statusCode: 'SUBMITTED', typeCode: 'AP_PSS' })
-    cy.task('stubGetLicencesToApprove')
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Now that we have the caseload on the backend, I don't believe the get functions for licences to approve or recently approved are needed anymore (added a TODO where they are used)?